include/arch/arm/aarch64/exc.h

```diff
@@ -26,7 +26,10 @@ extern "C" {
 
 struct __esf {
 	struct __basic_sf {
-		u64_t regs[20];
+		u64_t spsr;
+		u64_t elr;
+		u64_t pc;
```

Comment:
This is really only an an artifact of `x0` being the first argument to the thread entry function, which points to the actual thread function.

Under normal circumstances, `elr` is the `pc` of the interrupted context, so there need not be another PC. This should really be just `x0` or regs[0].

Reply:
I thought PC was a better suited name but I'll change it back (it was already regs[0])